packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -100,6 +117,14 @@ class Scaffold extends StatefulComponent {
 
 class ScaffoldState extends State<Scaffold> {
 
+  // DRAWER API
+
+  final GlobalKey<DrawerControllerState> _drawerKey = new GlobalKey<DrawerControllerState>();
+
+  void openDrawer() {
+    _drawerKey.currentState?.open();
```

Comment:
I think you should assert that you have one, rather than silently doing nothing if you don't.